.tests/formatting-sequentiality

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+from functools import reduce
+import sys
+import os
+
+rfcs = []
+for root, subdirs, files in os.walk("."):
+    dirs = root.split(os.path.sep)
+
+    # Skip the root directory
+    if len(dirs) < 2:
+        continue
+
+    # Skip hidden directories
+    if reduce(lambda x, y: x or y, map(lambda x: x.startswith('.'), dirs[1:]), False):
+        continue
+
+    # Append all files
+    rfcs += [(f, root) for f in files]
+
+status = 0
+for i, (name, root) in enumerate(sorted(rfcs)):
+    if i != int(name.split('-')[0]):
+        print("%s/%s is non-sequential!" % (root, name), file=sys.stderr)
```

Comment:
Would it make sense to add which number it should be, in case someone is confused?

Reply:
@lkatalin Fixed.